application/src/main/java/org/togetherjava/tjbot/features/moderation/TransferQuestionCommand.java

```diff
@@ -162,27 +160,28 @@ private RestAction<User> createForumPost(ModalInteractionEvent event, User origi
             .setTags(ForumTagSnowflake.fromId(defaultTag.getId()))
             .map(ForumPost::getMessage)
             .flatMap(message -> message.reply(originalUser.getAsMention()))
-            .map(sent -> originalUser);
+            .map(sent -> new PostEntity(originalUser, sent));
     }
 
-    private RestAction<Message> dmUser(MessageChannelUnion sourceChannel, User originalUser,
+    private RestAction<Message> dmUser(MessageChannelUnion sourceChannel, PostEntity postEntity,
```

Comment:
i dont like the variable name, to which post is it referring? the original message in chit_chat? the forum post message? sth else?

Reply:
Post is only referred to posts created in forum channels, text messages should be referred as messages.

Reply:
yeah, but which one is it? u could rename the variable to make it a bit clearer imo. `forumPost` perhaps 🤷 
dunno, it tripped me.